src/sidebar/services/groups.ts

```diff
@@ -163,11 +162,7 @@ export class GroupsService {
     // logging in or logging out.
     watch(
       this._store.subscribe,
-      () =>
-        /** @type {const} */ ([
-          this._store.hasFetchedProfile(),
-          this._store.profile().userid,
-        ]),
+      () => [this._store.hasFetchedProfile(), this._store.profile().userid],
```

Comment:
Should this be `[...] as const` or are the types of individual elements in the tuple preserved without that?

Reply:
Oh, I didn't realize that jsdoc was equivalent to `as const`. Since tests and linting was passing I completely forgot to come back at it.